### DISCUSSION TITLE
how might we set up separate github discussion thread for D&G?

### DISCUSSION THREAD

**Discussion by kjacobs01** [twilio-labs/paste#2718] [Office hours] [answered]
Hi there, 

Opening a thread to see how this process works and figure out how we might adapt it for D&G design system office hours. 

What would be the best way to do that?

Here is Brandon's WIP sign up sheet **EDITED TO REMOVE LINK**

**Comment by ghost** [comment, marked as answer]
Hi @kjacobs01 thanks for the discussion here. Just FYI I removed the link to protect any company information that may end up there into the future. This project and the discussions are open source!

Can you provide a few more specifics around what you're looking to learn or gather here? I can say that Github discussions are associated 1:1 with a repository. There is good documentation around [the tool and what it can and can't do here](https://docs.github.com/en/discussions).

**Reply by serifluous** [reply]
Update: Kelly and Brandon chatted about this in the #ux-infra-bpm-one-twilio channel, and we've added the "Office Hours: Data & Growth" category to try out using this forum for D&G migration support.
